Backend/Models/Database/OrderState.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+using Backend.Models.OrderEntity;
+
+namespace Backend.Models.Database
+{
+    public enum OrderStateId
+    {
+        Created = 1,
+        Approved,
+        InPreparation,
+        Ready,
+        InTransit,
+        Delivered,
+        OnTheWayToReturn,
```

Comment:
Maybe it would be simpler to call just 'Returning'?'

Reply:
fixed